src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -423,6 +450,22 @@ public void onWorldTick(TickEvent.WorldTickEvent event)
     }
 
     /**
+     * Updates all subscribers of fields etc.
+     */
+    private void markFieldsDirty()
+    {
+        ColonyManager.markDirty();
```

Comment:
Do we want to do a full markDirty? or does this just queue the dirty things to sync?